### ISSUE TITLE
Linenoise returned nil [IOError] / Allow Ctrl+D to quit the Interactive shell

### ISSUE THREAD

**Issue by kaushalmodi** [AndreiRegiani/INim#9]
IPython for example allows Ctrl + D to quit it.

Doing that in INim gives:

```text
INim 0.2.0
Nim Compiler Version 0.18.1 [Linux: amd64] at /home/kmodi/stowed/bin/nim
>>>
inim.nim(141)            inim
inim.nim(78)             runForever
rdstdin.nim(83)          readLineFromStdin
Error: unhandled exception: Linenoise returned nil [IOError]
```

**Comment by AndreiRegiani** [comment]
This is related to an open issue in Nim's stdlib module `rdstdin` (which uses `Linenoise` C library)
https://github.com/nim-lang/Nim/issues/5795

For a "clean" exit use the command `exit` or `quit()`.
Let's leave this issue opened until stdlib is fixed.

**Comment by AndreiRegiani** [comment]
I realized I could just handle the `IOError` exception, so it's all good for INim.
**Released v0.2.3** with this fix, please update with `nimble install inim`

Fix: https://github.com/AndreiRegiani/INim/commit/91c2d1a1e5292d8a8c37adf1f36326a181fbae7a

**Comment by kaushalmodi** [comment]
Awesome! Thank you, I confirm the fix.
